TIPs/TIP-27.md

```diff
@@ -0,0 +1,39 @@
+| id | Title | Status | Author | Description | Discussions to | Created |
+| ----------- | ----------- | ----------- | ----------- | ----------- | ----------- | ----------- |
+| TIP-27 | Thales Royale change contract| To be confirmed | gruja (@gruja.work) | Change of default position, sign up players on behalf of owners which did't position at first season first round | [discord url](https://discord.com/channels/906484044915687464/939095931490549800) | 2022-02-09
```

Comment:
Title should be more descriptive, e.g. Thales Royale default position on sign up